packages/commerce-sdk-react/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@salesforce/commerce-sdk-react",
-  "version": "1.1.0-dev.2",
+  "version": "1.0.1-preview.0",
```

Comment:
I'm now not sure whether this should be a patch or minor version. See https://github.com/SalesforceCommerceCloud/pwa-kit/pull/1371#discussion_r1269973490